charts/postgresql/values.yaml

```diff
@@ -0,0 +1,14 @@
+architecture: replication
+global:
+  storageClass: local-storage
+auth:
+  replicationPassword: externalPostgresql
+  postgresPassword: externalPostgresql
+  username: externalPostgresql
+  password: externalPostgresql
+  database: externalPostgresql
```

Comment:
Ugh. Can you remove the whole auth block? There should never be any default passwords in plaintext. The user of a helm chart should specify their own passwords. Same for database name and username. Please remove the whole auth block here.

Reply:
fixed